importers/importBasicCourseDetails.js

```diff
@@ -116,10 +116,6 @@ var importSubject = function (semester, subject) {
       process.stdout.write(' ' + courseData.catalog_number)
     }
 
-    if (typeof (courseData.catalog_number) === 'undefined' || courseData.catalog_number.length < 3) {
```

Comment:
I understand what you're doing here, @awonnacott, but we still need error-handling code to catch cases where the catalog number _is_ undefined or is shorter than three characters and not 90.

Could you re-submit with a conditional something along the lines of:
```javascript
if (typeof (courseData.catalog_number) === 'undefined' || courseData.catalog_number.length < 3 && courseData.catalog_number != 90)
```

I would do it myself, but I want to make sure you get credit for your contribution!

Reply:
Sure!

Reply:
How does it look now?

Reply:
bump